functions/index.js

```diff
@@ -25,4 +26,8 @@ module.exports = {
 
   /* Migration Scripts - For running  */
   ...migrationScripts,
+
+  /* sysconfig */
```

Comment:
remove comments

Reply:
I updated the cloud functions, this is an comment of what it refers to, do you still want to delete it?

Reply:
oh ok just remove it